src/test/java/io/nats/client/impl/JetStreamGeneralTests.java

```diff
@@ -801,107 +806,112 @@ public void testSubscribeDurableConsumerMustMatch() throws Exception {
             JetStream js = nc.jetStream();
             JetStreamManagement jsm = nc.jetStreamManagement();
 
-            createDefaultTestStream(jsm);
+            String stream = stream();
+            String subject = subject();
+            createMemoryStream(nc, stream, subject);
```

Comment:
Whenever I'm in tests now, I make them less generic, meaning I give unique values to things like stream and subject instead of using a fixed constant. There is no difference in test behavior, but this will allow me to re-use server instances in the future if necessary, as sometimes resources on build machines are low, and not having to start up a server every time would failures based on inability to start servers. The change will allow allow more parallelization of tests.